server/store/store.go

```diff
@@ -37,7 +40,16 @@ func openAdapter(workerId int, jsonconf json.RawMessage) error {
 	}
 
 	if adp == nil {
-		return errors.New("store: database adapter is missing")
+		if len(config.UseAdapter) > 0 {
+			// Adapter name specified explicitly.
```

Comment:
Maybe also for compatibility with the previous version check if `len(config.UseAdapter) == 1` and then just use it.

Reply:
Wait, but `len(config.UseAdapter)` returns the length of the string.
If you mean `len(config.Adapters) == 1` and using the only available adapter,
isn't that something we've discussed: https://github.com/tinode/chat/pull/377#discussion_r389472121

Reply:
Yes, I meant `len(config.Adapters) == 1`.

Apparently I misunderstood your code change. Sorry. I think automatically selecting the adapter when `len(config.Adapters) == 1` is a good thing. Otherwise we would force everyone to update their conf files. And the log entry is not needed. We are printing selected adapter in `main.go`.

Reply:
Done.